src/styles/index.scss

```diff
@@ -1,3 +1,100 @@
+@import 'fonts';
+@import 'stars';
+@import 'variables';
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: $fontRoboto;
+  font-size: 12px;
+  box-sizing: border-box;
+}
+
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding: 16px;
+  padding-top: 32px;
+  background-color: $basicWhite;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__img {
+    width: 160px;
+    max-width: 100%;
+    height: 135px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    margin: 0;
+    padding: 0;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 5px;
+    color: $mainAcsent;
+  }
```

Comment:
You've set margin and padding to 0 on lines 36 and 37. This is unnecessary because these properties don't have default values. You can safely remove these lines.